eng/Version.Details.xml

```diff
@@ -358,18 +358,18 @@
       <Uri>https://github.com/dotnet/runtime-assets</Uri>
       <Sha>99168dcff56809205e7ef8530d1256f3a07fab1f</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.8.0-3.23469.1">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.8.0-3.23471.2">
```

Comment:
Why are these being updated in this PR?

Reply:
I suspect the target branch change caused this: the PR was initially targeting release/8.0.. Good catch.

If you don't mind, can you push the right fix to this PR so that we can get this merged sooner, @radical?

Reply:
Actually hold on @radical - I think because the original branch itself was targeting release/8.0 it brought many more commits that are not wanted here, not just the ones you're pointing out. @pavelsavara can you please take care of it yourself? Reset your local branch to point to rc2, then cherry-pick your changes, and force push to this PR.

Reply:
excellent catch!